pkg/database/database.go

```diff
@@ -116,6 +116,7 @@ func Backup(backupPath string) error {
 }
 
 func RestoreFromBackup(backupPath string) error {
+	// TODO: should rename/move the backup, a restore operation should leave the backup intact.
```

Comment:
This method is used specifically when the migration process fails. We don't want to leave backup files around every time the migration fails. It should probably be documented as such, and future iterations of this method may behave differently if we expose this in the external API, but I don't think this TODO is valid.

Reply:
Sorry, i should have checked further. Just instinctual reaction to seeing a backup being moved. I've removed the comment.